.github/workflows/ci.yml

```diff
@@ -260,7 +260,7 @@ jobs:
     - name: Setup .NET Core SDK
       uses: actions/setup-dotnet@v2
       with:
-        dotnet-version: '6.0.x'
+        dotnet-version: '6.0.202'
```

Comment:
out of curiosity - this is required because ?

Reply:
a new version came out today and is causing the formatting break in the build release.
We decided to pin the release the version number.

Reply:
b/c it changed today and is breaking the build. I suggested win Pin it until were ready to update it and reformat files